pkg/resources/sku/sku.go

```diff
@@ -190,15 +198,16 @@ func (p AProductConfig) mutate(podTemplateSpec *v13.PodTemplateSpec, name string
 }
 
 func (p AProductConfig) mutateResources(pod, cfg v13.ResourceList) {
-
 	podcpu := pod[v13.ResourceCPU]
 	//Cmp returns -1 if the quantity is less than y (passed value) so if podcpu is less than cfg cpu
 	if p.sku.isUpdated || podcpu.Cmp(cfg[v13.ResourceCPU]) == -1 || podcpu.IsZero() {
-		pod[v13.ResourceCPU] = cfg[v13.ResourceCPU]
+		quantity := cfg[v13.ResourceCPU]
+		pod[v13.ResourceCPU] = resource.MustParse(quantity.String())
```

Comment:
Getting a nil pointer here on a fresh install possibly due to pod can be nil :thinking: